GameSessionsManager.h

```diff
@@ -1,19 +1,25 @@
 #ifndef TICTACTOE_GAMESESSION_H
 #define TICTACTOE_GAMESESSION_H
 
-#include "SearchManager.h"
 #include <thread_pool.h>
+#include <GrpcDelegatingWriter.h>
+#include <GrpcDelegatingReader.h>
+#include <Player.h>
+#include "PlayerInfo.h"
 #include <GameSession.h>
 
 class GameSessionsManager {
-	ThreadPool threadpool;
+	ThreadPool game_session_threads;
 
-public:
+	Player create_player(const std::string& player_id, const symbol& symbol_, const std::string& player_address){
```

Comment:
What is this factory method for? And `symbol` is an enum. Passing it by value or const reference doesn't make much a difference in terms of memory